app/views/partials/_resource_list.erb

```diff
@@ -0,0 +1,50 @@
+<div class="row">
+  <div class="col-xs-12">
+    <h3>Currently Avaliable Resources</h3>
+  </div>
+</div>
+
+<div class='display' class="row">
+  <div class="col-xs-4">
+    <div class="panel panel-default">
+      <div class="panel-heading">Trucks</div>
+      <% if trucks.length == 0 %>
+        <h5>No trucks currently avaliable</h5>
+      <%else%>
+        <% trucks.each do |truck| %>
+          <div class="panel-body">
+            <%= "#{truck.name} - #{truck.model}" %>
```

Comment:
extract this out to a helper so you can reuse it and have less logic/formatting in your views